ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-pim-sparse-mode.j2

```diff
@@ -17,25 +17,40 @@
 {%                 endfor %}
 {%             endfor %}
 {%         endif %}
-{%         if router_pim_sparse_mode.ipv4.anycast_rps is defined and router_pim_sparse_mode.ipv4.anycast_rps is not none %}
-{%         endif %}
-{%         if router_pim_sparse_mode.ipv4.anycast_rps is defined and router_pim_sparse_mode.ipv4.anycast_rps is not none %}
+{%         if router_pim_sparse_mode.ipv4.anycast_rps is arista.avd.defined %}
 
 ### IP Anycast Information
 
 | IP Anycast Address | Other Rendez-vous Point Address | Register Count |
 | ------------------ | ------------------------------- | -------------- |
 {%             for anycast_rp in router_pim_sparse_mode.ipv4.anycast_rps | arista.avd.natural_sort %}
 {%                 for other_anycast_rp_address in router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses | arista.avd.natural_sort %}
-| {{ anycast_rp }} | {{ other_anycast_rp_address }} | {% if router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count is defined and router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count is not none %} {{ router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count }} | {% else %} - {% endif %}|
+| {{ anycast_rp }} | {{ other_anycast_rp_address }} | {% if router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count is arista.avd.defined %} {{ router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count }} | {% else %} - {% endif %}|
 {%                 endfor %}
 {%             endfor %}
 {%         endif %}
+{%     endif %}
+
+{%     if router_pim_sparse_mode.vrfs is arista.avd.defined %}
+
+### IP Sparse Mode VRFs
+
+| VRF Name | Rendez-vous Point Address | Group Address |
+| -------- | ------------------------- | ------------- |
+{%         for vrf in router_pim_sparse_mode.vrfs | arista.avd.natural_sort('name') %}
+{%             if vrf.ipv4.rp_addresses is arista.avd.defined %}
+{%                 for rp_address in vrf.ipv4.rp_addresses | arista.avd.natural_sort('address') %}
+{%                     for group in rp_address.groups | arista.avd.natural_sort %}
+| {{ vrf.name }} | {{ rp_address.address }} | {{ group }} |
+{%                     endfor %}
```

Comment:
Add all groups in the same row. Please change the lines 43-45 to:

```
| {{ vrf.name }} | {{ rp_address.address }} | {{ rp_address.groups | join(', ') }} |
```

Reply:
Implemented as requested